mvp-nullobject-presenter/src/main/resources/META-INF/proguard/mvp-nullobject-presenter.pro

```diff
@@ -0,0 +1 @@
+-keep interface * extends com.hannesdorfmann.mosby3.mvp.MvpView
```

Comment:
Name of this class is not important, so adding `allowobfuscation` should be fine.